examples/chip-tool/commands/common/Command.cpp

```diff
@@ -87,6 +87,7 @@ static bool ParseAddressWithInterface(const char * addressString, Command::Addre
 bool Command::InitArgument(size_t argIndex, char * argValue)
 {
     bool isValidArgument = false;
+    bool isHexNotation   = strncmp(argValue, "0x", 2) == 0 || strncmp(argValue, "0X", 2) == 0;
```

Comment:
Curious why we are not using std::string for this if we are using string streams anyway. With std::string, it would be `xxx.substr(0, 2) == "0x"` which is a bit friendlier. It's too bad `.starts_with()` is C++20 :(